src/main/java/com/github/fge/jsonpatch/AddOperation.java

```diff
@@ -74,7 +76,7 @@
     public AddOperation(@JsonProperty("path") final JsonPointer path,
         @JsonProperty("value") final JsonNode value)
     {
-        super("add", path, value);
+        super(OPERATION_NAME, path, value);
```

Comment:
Do we still need this OPERATION_NAME in the parent constructor now that it is in the factory?